cmd/object.go

```diff
@@ -77,7 +77,7 @@ var getObjectPropertiesCmd = withLocalFlags(&cobra.Command{
 		state := internal.PrepareEngineState(rootCtx, headers, tlsClientConfig, false, false)
 		printer.PrintGenericEntityProperties(state, args[0], "object", viper.GetBool("minimum"))
 	},
-}, "minimum")
+}, "minimum", "full")
```

Comment:
What happens if both minimum and full is set?
Right now it seems like minimum will have precedence.